udata/core/contact_point/api.py

```diff
@@ -0,0 +1,68 @@
+from udata.api import api, API
+from udata.api.parsers import ModelApiParser
+
+from .api_fields import contact_point_fields, contact_point_page_fields
+from .forms import ContactPointForm
+from .models import ContactPoint
+
+
+class ContactPointApiParser(ModelApiParser):
+    sorts = {}
+
+    def __init__(self):
+        super().__init__()
+
+
+ns = api.namespace('contacts', 'Contact points related operations')
+
+contact_point_parser = ContactPointApiParser()
+
+
+@ns.route('/', endpoint='contact_points')
+class ContactPointsListAPI(API):
+    '''Contact points collection endpoint'''
+    @api.doc('list_contact_points')
+    @api.marshal_with(contact_point_page_fields)
+    def get(self):
+        '''List all contact points'''
```

Comment:
I wonder if this list makes much sense? In particular since we don't have orga/owner in resulting fields